src/librustc_typeck/diagnostics.rs

```diff
@@ -4567,6 +4567,52 @@ i_am_a_function();
 ```
 "##,
 
+E0619: r##"
+A not (yet) known type was used.
```

Comment:
The first sentence isn't supposed to be an explanation, just a short recap of the error. However this explanation could replace the current one since it's more complete.